adguard/config.json

```diff
@@ -14,11 +14,13 @@
   "init": false,
   "ports": {
     "53/udp": 53,
-    "80/tcp": null
+    "80/tcp": null,
+    "853/tcp": null
```

Comment:
AdGuard runs on the host network...

Reply:
was one of my first questions :) 
Does it need to be setup in the addon as portainer doesn't show any portforwarding setup in the addon container.

i was using 853 on my portscanner instead of 583 (was time to go to bed), sorry from my end !

i can confirm you don't need to setup forwarding as the port shows as open and container is using host network. New to the whole addon thing so wasn't sure something went different than normal in deploying containers.

Reply:
oops...